src/utils/cid/mod.rs

```diff
@@ -63,13 +105,27 @@ mod tests {
     use crate::utils::db::CborStoreExt;
     use anyhow::*;
     use cid::{
-        multihash::{Code, MultihashDigest},
+        multihash::{self, Code, MultihashDigest},
         Cid,
     };
     use fvm_ipld_encoding::DAG_CBOR;
+    use quickcheck::Arbitrary;
     use quickcheck_macros::quickcheck;
     use std::mem::size_of;
 
+    impl Arbitrary for CidVariant {
+        fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+            // Quickcheck does not reliably generate the DAG_CBOR/Blake2b variant of V1 CIDs, but we can manually create them. This method will generate each variant of `CidVariant` with equal probability.
```

Comment:
This comment is not correct. It's only when generating a random `Cid` that it is unlikely to be a v1-dag_cbor-blake2b variant.

Reply:
Changed to derive `Arbitrary`, so removed custom impl and associated comment.